spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -51,12 +51,13 @@ interface FungibleTokenPacketDataV2 {
   sender: string
   receiver: string
   memo: string
+  forwardingPath: []string // a list of portID/channelID pairs determining where the tokens must be forwarded next
 }
 
 interface Token {
   denom: string // base denomination
   trace: []string
```

Comment:
it would be helpful if maybe we clarified the naming here, between `trace` and `path` -- below we understand that traces are only the port and channel pairs, but in ie: the `DenomTrace` struct we understand that concept of trace as containing the path AND base denom.

Reply:
there's also quite a lot of overlap between DenomTrace and Token, the only difference seems to be that the Token trace is structured a bit differently than the trace on DenomTrace (port/channel pairs v one long string), and that there's an amount field. would we maybe want to embed the DenomTrace struct into Token and then have those methods be available there?

Reply:
Thanks @charleenfei for raising this concern. The approach you propose is pretty similar to the one that our colleagues at informal systems have used to model the ics20v1 in quint. I personally like it.